### DISCUSSION TITLE
Null resolutions

### DISCUSSION THREAD

**Discussion by aaronadamsCA** [yarnpkg/berry#4794] [Ideas]
In OSS it's still fairly common for big projects to carry around direct dependencies that you know you don't actually need. Some of these dependencies can themselves have transitive dependencies that are large in number or very outdated.

I've found a simple enough solution in our monorepos:

```json
  "resolutions": {
    "@storybook/builder-webpack4": "npm:empty-npm-package@1.0.0",
    "@storybook/manager-webpack4": "npm:empty-npm-package@1.0.0",
    "eslint-plugin-graphql": "npm:empty-npm-package@1.0.0"
  },
```

Of course, this can confuse peer dependencies, for one, since now Yarn internally views this package as `v1.0.0`. There might be other gaps, too.

I could create local packages with a `package.json` with just `{name: "@storybook/builder-webpack4"}`, and then point the resolution to them with `"workspace:*"`, but this requires a bunch of stub packages.

If there was just some way to provide a null resolution - effectively resolving any package to an empty package - that would be a great thing to have for this.

**Comment by RDIL** [comment]
I made a plugin for this a while back.  Let me try to find it for you.
